src/Servers/IIS/IIS/samples/NativeIISSample/NativeIISSample.csproj

```diff
@@ -5,7 +5,6 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <InProcessTestSite>true</InProcessTestSite>
-    <AspNetCoreHostingModel>inprocess</AspNetCoreHostingModel>
```

Comment:
@pranavkm if this is now the default then does the other sample need to be updated for out-of-proc?

Reply:
Did you mean @pakrym ?

Reply:
Do we have any samples that explicitly rely on running out-of-proc?

Reply:
@Tratcher that shouldn't stop this PR right? It's just a followup item for @pakrym.

Reply:
This one: https://github.com/aspnet/AspNetCore/blob/b5f21b42be204f6c0a97a2f12bdfeef5d9be3759/src/Servers/IIS/IISIntegration/samples/IISSample/IISSample.csproj